package.json

```diff
@@ -52,7 +52,7 @@
     "aigle": "^1.14.1",
     "aware": "^0.3.1",
     "bignumber.js": "^9.0.0",
-    "dvf-utils": "https://github.com/DeversiFi/dvf-utils.git#bdcf7b9699831d3101a05a79014e0b2de11ce6b0",
+    "dvf-utils": "https://github.com/DeversiFi/dvf-utils.git",
```

Comment:
I think we should use a fixed commit for dvf-utils to avoid accidentally introducing changes at different times on the backend and frontend